alacritty_terminal/src/input.rs

```diff
@@ -392,7 +395,10 @@ impl Action {
                 ctx.spawn_new_instance();
             },
             Action::None => (),
+            Action::ReceiveChar => return false
```

Comment:
We should also swap these (None/ReceiveChar) match arms. To be consistent with already suggested changes on swapping things.

Reply:
With my suggested changes, these actions would just be merged together anyways.

Reply:
I see, I've suggested it, since `return false` was presented in your diff. Anyway it should be `Action::ReceiveChar | Action::None => ...`  (not None | ReceiveChar) , right?

Reply:
Yeah, I'd go for that. Though I don't particularly care a lot about the order when it's in the same statement. That's a bit too nitpicky even for me. :)